kong/plugins/aws-lambda/schema.lua

```diff
@@ -1,3 +1,14 @@
+local function check_status(status)
```

Comment:
We also need a test for this validation function similar to the `schema_spec` tests you can found in other parts of the unit test suite.